### PR TITLE
Some work on Netflix quality checks

### PR DESCRIPTION
**Pull request by OmrSi**
1. Set CPS for various languages.
2. Include all 4 dialog styles and set the correct ones for various languages.
3. Add an option to set CPS for Children programs.
4.  Add an option to set CPS for SDH subtitles.
5. Add an option for `When including ellipses in subtitles, please use the single smart character (U+2026) as opposed to three dots/periods in a row.`
6. Make the gridlines depend on `DarkThemeShowListViewGridLines`.
7. Hide the "Apply" column for now.

Closes: https://github.com/SubtitleEdit/subtitleedit/issues/4699

### CODE REVIEW

**Comment by OmrSi** [comment]
I don't know if it's better to add `IsChildrenProgram` as a property to `NetflixQualityController` or as a field to `NetflixCheckMaxCps`, I did the latter, tell me if the former is better.

**Comment by OmrSi** [comment]
@niksedk Should the `NetflixQualityCheck` folder be moved to `Forms` in `LibSE`?

**Comment by Flitskikker** [comment]
> 3. Add an option to set CPS for Children programs, which makes it CPS - 3.

Maybe add support for actual values? For Dutch, it's CPS - 4:
- Adult programs: 17 characters per second
- Children’s programs: 13 characters per second

Maybe also add an option for "Subtitles for the Deaf and Hard of Hearing (SDH)"?
Then, for Dutch content, the values are:
- Adult programs: 20 characters per second
- Children’s programs: 17 characters per second

**Comment by Flitskikker** [comment]
Actually, these values* seem to apply for most languages (not only Dutch):
https://partnerhelp.netflixstudios.com/hc/en-us/sections/203480497?page=1#articles

*)
Regular subtitles:
- Adult programs: 17 characters per second
- Children’s programs: 13 characters per second

Subtitles for the Deaf and Hard of Hearing (SDH):
- Adult programs: 20 characters per second
- Children’s programs: 17 characters per second

**Comment by OmrSi** [comment]
@Flitskikker How is it now?

File: src/libse/NetflixQualityCheck/NetflixQualityController.cs [hunk 1]
```diff
@@ -54,6 +121,41 @@ public int SingleLineMaxLength
...
+                    case "pl": // Polish
+                    case "ro": // Romanian
+                    case "ru": // Russian
+                    case "sk": // Slovak
+                    case "es": // Spanish
+                    case "th": // Thai
+                    case "vi": // Vietnamese
+                        return DialogType.DashBothLinesWithSpace;
+                    case "nl": // Dutch
+                    case "fo": // Finnish
```
Comment (Flitskikker) [inline src/libse/NetflixQualityCheck/NetflixQualityController.cs:147]: Shouldn't this be `fi`?

**Comment by Flitskikker** [comment]
@OmrSi 
Looking good for the rest!

File: src/libse/NetflixQualityCheck/NetflixQualityController.cs (hunk 1 above)
Comment (OmrSi) [reply src/libse/NetflixQualityCheck/NetflixQualityController.cs:147]: Oh, yeah. I got confused and took the first result:
image

I mean, I just want Finnish, why show me the rest? xD

**Comment by OmrSi** [comment]
@niksedk This is ready now.

Should the `NetflixQualityCheck` folder be moved to `Forms` in `LibSE`?

**Comment by niksedk** [comment]
> 
> 
> @niksedk This is ready now.
> 
> Should the `NetflixQualityCheck` folder be moved to `Forms` in `LibSE`?

No, I don't think so...

**Pull request merged**